### PR TITLE
HW 5

### PR DESCRIPTION
**Pull request by AlinaZvarych**

### CODE REVIEW

File: src/main/java/hw_5/myCity/Building.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package hw_5.myCity;
+
+import hw_5.myClass.Person;
+
+public class Building {
+    Person[] residents;
+
+    public Building(int numberOfResidents) {
+        residents = new Person[numberOfResidents];
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_5/myCity/Building.java]: You don't need these elements. 

Screenshot_5

just create a main method inside Building and inside main method create Persons and add persons to the array. Using this array display persons

File: src/main/java/hw_5/Point.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package hw_5;
+
+public class Point {
+    int x;
```
Comment (AntBelyi) [inline src/main/java/hw_5/Point.java]: access modifier?

File: src/main/java/hw_5/Point.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    int y;
```
Comment (AntBelyi) [inline src/main/java/hw_5/Point.java]: access modifier?

File: src/main/java/hw_5/myCity/Building.java (hunk 1 above)
Comment (AlinaZvarych) [reply src/main/java/hw_5/myCity/Building.java]: done

File: src/main/java/hw_5/myClass/Person.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package hw_5.myClass;
+
+public class Person {
+    public String name;
+    public String surname;
+    public int age;
```
Comment (AntBelyi) [inline src/main/java/hw_5/myClass/Person.java:6]: Are you sure it is a good approach to use a public modifier?

File: src/main/java/hw_5/Point.java (hunk 2 above)
Comment (AlinaZvarych) [reply src/main/java/hw_5/Point.java]: done

File: src/main/java/hw_5/Point.java (hunk 3 above)
Comment (AlinaZvarych) [reply src/main/java/hw_5/Point.java]: done

**Pull request merged**